django/utils/html.py

```diff
@@ -357,7 +357,9 @@ def handle_word(
                     domain = punycode(domain)
                 except UnicodeError:
                     return word
-                url = self.mailto_template.format(local=local, domain=domain)
+                url = self.mailto_template.format(
+                    local=quote(local, safe=""), domain=quote(domain, safe="")
+                )
```

Comment:
I can update this to:
```suggestion
                local = quote(local, safe="")
                url = self.mailto_template.format(local=local, domain=domain)
```
without a test failure, do we need the quoting on the domain? And if so, can you add a test for this?

Reply:
If domain is a valid, ASCII-only domain, quoting it is unnecessary. And the call to punycode() just above should ensure it's ASCII-only. So your suggestion seems reasonab…

… oh, hmm, oops:

```python
urlize("test@example.com?org")
'<a href="mailto:test@example.com?org">test@example.com?org</a>'
```

I'll add another test case for this PR, and I guess open a new ticket. \<sigh>

Reply:
Thank you :+1: